inst/rmarkdown/templates/template_sfp_en/skeleton/skeleton.Rmd

```diff
@@ -10,28 +10,44 @@ params:
   version_number: "{{{version_number}}}"
   language: {{{language}}}
   theme: {{{theme}}}
+  dependencies_protocolcode: [sfp_4##_en, sfp_3##_en]
+  dependencies_versionnumber: [YYYY.NN, YYYY.NN]
+  dependencies_params: ["list(width_m = 4, length_m = 4)", "list()"]
+  dependencies_appendix: [TRUE, FALSE]
 site: bookdown::bookdown_site
 output: 
   bookdown::gitbook:
     split_by: none
     split_bib: FALSE
 ---
 
+
 <!--
+Instructies bij declaratie van dependencies in de yaml header:
+Er zijn vier velden in de yaml header die je moet aanpassen voor project-specifieke protocols:
```

Comment:
> voor project-specifieke protocols

I see you implement the facility both for sfp and spp, I think that's OK (an sfp could also require a variant of another protocol), but best generalize the wording here (you refer to spp while this is the skeleton of sfp)

Reply:
Extending on that, the sfp template currently has no `08_subprotocols.Rmd` file.

Reply:
The documentation needs improvement, but some discussion is needed first.

My idea is that sfp protocols can have dependencies, but that these dependencies are not included as subprotocols. That is why I have not included that PART division chapter. If a sfp depends on a protocol, and in turn that sfp is a dependency of an spp, we will have a subprotocol chapter with another subprotocol nested in it (and possibly deeper nesting).  Do we want to allow such complexity?

I believe (hope) dependencies of sfp protocols will not be parameterized protocols. I image sfp's might depend on sip's, but not on other sfp's.  In that case, we have no real need to render the dependency as a subprotocol (we can just refer to the specific version of it when there are no variants)

Reply:
> My idea is that sfp protocols can have dependencies, but that these dependencies are not included as subprotocols. That is why I have not included that PART division chapter.

I can follow that, but note that the sfp `skeleton.Rmd` currently also provides the `dependencies_params` key.

>  a subprotocol chapter with another subprotocol nested in it (and possibly deeper nesting)

But I wouldn't have a problem if it already works out of the box, flexibility is a bonus IMO. But I would not actually invest more effort to achieve that. Anyway, `knit_child()` can be used in a nested way; already done that. If nestedness is actually already supported, it would mean the user would need to pass a `dependencies_params` parameter to the subprotocol I guess :wink:

Reply:
This is now implemented by 7c0fd4d